.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
```

Comment:
I would add here one additional point:
 ** Please describe the problem that you would like to solve with this feature **
Which problem do you want to have solved and how is this related to IDS and the work of IDSA.

Reply:
Thank you for the comment. I replaced the first question with this one, to refrain from having two similar questions.